### ISSUE TITLE
Letsencrypt for Route 53 root domain and wildcard

### ISSUE THREAD

**Issue by themsaid** [AnalogJ/lexicon#281]
If I generate a certificate for root domain only or wildcard only it works, but if I generate a certificate for both I get this error.

<img width="1147" alt="screen shot 2018-08-29 at 7 38 06 pm" src="https://user-images.githubusercontent.com/4332182/44804545-e7c0a500-abc1-11e8-9ef0-5197e9ff9550.png">

**Comment by AnalogJ** [comment]
From your image its a bit difficult to determine exactly what the cause is. 

There are a couple of lexicon providers that need to be changed to support record sets, but all of the Certbot integrated providers should work out of the box.
You may need to bring up this issue with Certbot or the ACME client that you use. 

Which DNS provider are you attempting to communicate with? And which ACME/letsencrypt client are you using?

**Comment by themsaid** [comment]
I'm using Route53 as the DNS provider and dehydrated as the acme client, here's the hook registered: https://raw.githubusercontent.com/AnalogJ/lexicon/master/examples/dehydrated.default.sh

**Comment by taylorotwell** [comment]
The issue seems to be Route 53 doesn't like multiple TXT values with the same domain. I can't even do it manually in the Route 53 web console... even though I can do it in other providers like Cloudflare. 

So, that seems to be the problem.

**Comment by taylorotwell** [comment]
This SO issue seems to indicate the values have to all be added at the same time:

https://serverfault.com/questions/616407/tried-to-create-2-record-set-type-txt-in-route53

🤷‍♂️

**Comment by AnalogJ** [comment]
Yeah, currently the Route53 provider does not support record sets, theres an open issue for it: https://github.com/AnalogJ/lexicon/issues/262

I don't use Route53 with lexicon so it's a bit hard for me to test changes. If you'd be willing to open a PR there's an existing record-set test suite that you can use to verify your work. 

We tracked some of the providers that have record-set support in https://github.com/AnalogJ/lexicon/pull/190 so you can use those providers as examples if you do end up writing a PR.

**Comment by baochungit** [comment]
@AnalogJ Route 53 does support record sets. Take a look at bellow of value field, you will see a note "IPv4 address. Enter multiple addresses on separate lines.". However lexicon doesn't do it the same way.

**Comment by baochungit** [comment]
https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/ResourceRecordTypes.html#TXTFormat
Could lexicon support this soon?
